XenAdmin/SettingsPanels/VMHAEditPage.cs

```diff
@@ -421,18 +421,18 @@ public void SetXenObjects(IXenObject orig, IXenObject clone)
 			Repopulate();
 			haNtolIndicator.Connection = vm.Connection;
 			haNtolIndicator.Settings = Helpers.GetVmHaRestartPriorities(vm.Connection, Properties.Settings.Default.ShowHiddenVMs);
-			haNtolIndicator.UpdateInProgressChanged += new EventHandler(haNtolIndicator_UpdateInProgressChanged);
+			haNtolIndicator.UpdateInProgressChanged += haNtolIndicator_UpdateInProgressChanged;
 
 			// Put property changed listener on pool, disable edits if HA becomes disabled
 			pool = Helpers.GetPoolOfOne(vm.Connection);
 			if (pool != null)
 			{
 				pool.PropertyChanged += pool_PropertyChanged;
-			}
-			origNtol = pool.ha_host_failures_to_tolerate;
+                origNtol = pool.ha_host_failures_to_tolerate;
+            }

```

Comment:
It's because VS2019 by default uses spaces instead of tabs, which is more reliable (since it's always rendered the same).
@kc284 mentioned we prefer spaces for that reason

Reply:
As mentioned in a private chat, I will be updating the entire file to be consistent.